app/controllers/api/stateless/members_controller.rb

```diff
@@ -6,6 +6,36 @@ class MembersController < StatelessController
       def show
         @current_member
       end
+
+      def update
+        permitted_params = params.require(:member).permit(:first_name, :last_name, :email, :country, :city, :postal, :address1, :address2)
```

Comment:
ok to make `permitted_params` a private method? It'll make `update` less cluttered.